docs/robots/ridgeback/user_manual_ridgeback.mdx

```diff
@@ -1,12 +1,330 @@
+this cannot be changed since the MCU is statically set to 192.168.131.2 and expects the computer to be at
+192.168.131.1. Avoid IPs in use by the following pre-existing devices:
+
+| IP Address     | Device Assignment                              |
+| :------------- | :--------------------------------------------- |
+| 192.168.131.1  | Onboard PC (both ports, br0 network interface) |
+| 192.168.131.2  | Ethernet-connected MCU                         |
+| 192.168.131.20 | Front-facing LIDAR                             |
+| 192.168.131.21 | Rear-facing LIDAR (if present)                 |
+
+#### Payload Integration Plate
+
+The Payload Integration Plate provides a robust foundation for mounting payload structures such as
+UR5/UR10 manipulation arms or any other combination of sensors and manipulators. For more information
+and guidance on mounting payload structures on top of Ridgeback, please refer to
+[Mechanical Integration](/docs/robots/ridgeback/integration_ridgeback#ridgeback-payload-mechanical-integration).
+
+#### Orientation References
+
+The reference frame used by all Clearpath Robotics ground robots is based on ISO 8855.
+
+<!--- TODO:
+, and is shown in the figure below.
+Husky's front is shown.
+--->
+
+When commanded with a positive translational velocity (forward), wheels travel in the positive x-direction.
+The direction of the axes differs from those used for roll, pitch, and yaw in aircraft, and care should be taken to ensure that data is interpreted correctly.
+
+<!-- TODO: need image -->
+
+### System Specifications
+
+| Specification         | Measurement                                                                  |
+| :-------------------- | :--------------------------------------------------------------------------- |
+| Dimension, Length     | 960 mm                                                                       |
+| Dimension, Width      | 793 mm                                                                       |
+| Dimension, Height     | 311 mm <sup>†</sup>                                                          |
+| Ground Clearance      | 18 mm                                                                        |
+| Maximum Payload       | 100 kg                                                                       |
+| Speed, Maximum        | 1.1 m/s                                                                      |
+| Operating Environment | Indoor                                                                       |
+| Operating Time        | 15 hours with max payload (24 V 100 Ah)                                      |
+| Battery Chemistry     | Marine Grade AGM Sealed Lead Acid                                            |
+| Charge Time           | 8 hours                                                                      |
+| User Power            | 7.5 A @ VBAT (24 V), 7.5 A @ 12 V, 7.5 A @ 5 V                               |
```

Comment:
5 A for each of the rails, based off our VKS instructions of what fuses are installed.

Reply:
Updated